stingray/crossspectrum.py

```diff
@@ -121,6 +130,17 @@ def _make_crossspectrum(self, lc1, lc2):
         if not isinstance(lc2, lightcurve.Lightcurve):
             raise TypeError("lc2 must be a lightcurve.Lightcurve object")
 
+        # Then check that GTIs make sense
+        if self.gti is None:
+            self.gti = cross_two_gtis(lc1.gti, lc2.gti)
+
+        if self.gti.shape[0] != 1:
```

Comment:
I'm being strict on purpose :). If they supply GTIs, they should be in the right format, in my opinion.
I guess it would not be impossible to put a check on the shape of the array and transparently transform it. I'm not a big fan of silent transformations though. What do you think?